Tiltfile

```
@@ -0,0 +1,29 @@
+load('ext://restart_process', 'docker_build_with_restart')
+
+local_resource(
+    'gitops-bin', 
+    'make bin', 
```

```suggestion
    'GOOS=linux GOARCH=amd64 make bin', 
```

This will make this work on any platform.